src/main/java/io/ticktok/rest/TicktokRestClient.java

```diff
@@ -0,0 +1,50 @@
+package io.ticktok.rest;
+
+import org.apache.http.HttpHeaders;
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpEntityEnclosingRequestBase;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.message.BasicHeader;
+
+import java.io.IOException;
+
+public class TicktokRestClient {
+    private HttpClient client;
+    private final String domain;
+
+    public TicktokRestClient(String domain){
+        this.domain = domain;
+        this.client = HttpClientBuilder.create().build();
+    }
+
+    private void handleBodyCall(HttpEntityEnclosingRequestBase request, String entry) throws IOException {
+        request.setEntity(new StringEntity(entry));
```

Comment:
you can use the simpler Request.post, look in AppDriver in ticktok tests

Reply:
for now it works for me the use of `org.apache.httpcomponents.httpclient`

Reply:
Request.post is a simpler API from the same library

Reply:
maybe, but once ill add other types of requests, i need very little code to add.